bin/init/sh

```diff
@@ -0,0 +1,424 @@
+#!/usr/bin/env sh
+set -o errexit -o nounset -o xtrace
+
+PACKAGE_NAME="${PACKAGE_NAME:?}"
+package_dir="./packages/$PACKAGE_NAME"
+package_full_name="@utofun/$PACKAGE_NAME"
+
+sash_key_down='\033[B'
+
+sash_dirty_workspace() {
```

Comment:
I have a little question, what does "sash" mean here?

Reply:
It's the library name. Sash is like belt, a belt is for utility belt.